src/PostList.js

```diff
@@ -0,0 +1,16 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import Post from './Post';
+
+const PostList = props => (
+  <dl>
+    {props.posts.map(post => (
+      <Post post={post} key={post.id} />
```

Comment:
```suggestion
      <Post post={currentPost} key={post.id} />
```
три слова post подряд выглядит не очень, не понятно какой post за что отвечает. Давай разделять как-то назавния